editor/gl/program.ts

```diff
@@ -58,7 +65,9 @@ export class Program {
         for (let i = 0; i < nb_uniforms; ++i) {
             let uniform_info = gl.getActiveUniform(program, i);
             if (uniform_info) {
-                let loc = gl.getUniformLocation(program, uniform_info.name);
+                // This can't be null as the uniform_info is provided by
+                // the gl context. So we assert that.
```

Comment:
Is there an `assert` mechanism similar to the one in C? If there is, maybe it is a good idea to use it here, rather than just a command (it would be easier to understand a runtime error with a proper assertion the day the assumptions of gl context change)